src/SprykerEco/Zed/Afterpay/Persistence/Propel/Schema/spy_afterpay.schema.xml

```diff
@@ -48,20 +46,31 @@
 
     <table name="spy_payment_afterpay_transaction_log">
         <column name="id_payment_afterpay_transaction_log" type="INTEGER" primaryKey="true" autoIncrement="true"/>
-        <column name="fk_sales_order" required="true" type="INTEGER"/>
+        <column name="order_reference" required="true" type="VARCHAR"/>
         <column name="transaction_type" required="false" type="VARCHAR"/>
         <column name="outcome" required="true" type="VARCHAR"/>
         <column name="request_payload" required="false" type="LONGVARCHAR"/>
         <column name="response_payload" required="true" type="LONGVARCHAR"/>
 
-        <foreign-key name="spy_afterpay_transaction_log-fk_sales_order" foreignTable="spy_sales_order">
-            <reference foreign="id_sales_order" local="fk_sales_order"/>
-        </foreign-key>
+        <unique name="spy_payment_afterpay_transaction_log-order_reference">
+            <unique-column name="order_reference"/>
+        </unique>
```

Comment:
Clarification needed: `order_reference` should be `unique` in `transaction_log`?  So if one transaction is logged, another cannot be logged anymore?